sympy/rubi/sympy2matchpy.py

```diff
@@ -0,0 +1,38 @@
+'''
+Contains Sympy to MatchPy converter function.
+'''
+
+import matchpy
```

Comment:
You'll need to move matchpy imports inline so that they are only imported in Python 3.6.